### DISCUSSION TITLE
ValueError: Unable to configure formatter 'airflow'

### DISCUSSION THREAD

**Discussion by Clint-Os** [apache/airflow#41095] [General]
I'm getting this error and I cannot seem to find the solution or the cause at least. I have tried prompting gpt and see what I can do but to no avail, is there anyone who can help? Or has anyone encountered this issue?

**Comment by potiuk** [comment]
https://dontasktoask.com/

**Reply by potiuk** [reply]
Just describe in detail what your issue is, that increases your chances of getting help.
